cst/plugins/eu.esdihumboldt.cst.functions.numeric/src/eu/esdihumboldt/cst/functions/numeric/sequentialid/SequentialID.java

```diff
@@ -60,6 +59,10 @@ protected Object evaluate(String transformationIdentifier, TransformationEngine
 		String sequenceStr = getOptionalParameter(PARAM_SEQUENCE, Value.of(Sequence.type.name()))
 				.as(String.class);
 
+		// get starting value for the sequence - default is 1
+		String startValue = "1";
+		startValue = getOptionalParameter(START_VALUE, Value.of("")).as(String.class);
```

Comment:
If you use a default of an empty string, the default value will not be one, but cause an error (because an empty string cannot be converted to an integer later on).

Reply:
How do you think this was resolved?

I would suggest something like:

```
String startValue = getOptionalParameter(START_VALUE, Value.of("1")).as(String.class);
```